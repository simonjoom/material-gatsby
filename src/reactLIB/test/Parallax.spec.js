import React from 'react';
import { shallow } from 'enzyme';
import Parallax from '../src/Parallax';
import mocker from './helper/mocker';

describe('<Parallax />', () => {
  let wrapper;
  const parallaxMock = jest.fn();
  const restore = mocker('parallax', parallaxMock);

  afterAll(() => {
    restore();
  });

  test('should render a Parallax', () => {
    wrapper = shallow(<Parallax />);
    expect(wrapper).toMatchSnapshot();
  });
});
