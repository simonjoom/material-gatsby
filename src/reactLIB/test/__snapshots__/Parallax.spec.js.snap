// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Parallax /> should render a Parallax 1`] = `
<div
  className="parallax-container"
>
  <div
    className="parallax"
  >
    <img />
  </div>
</div>
`;
